projects/igniteui-angular/src/lib/list/list.common.ts

```diff
@@ -1,7 +1,7 @@
 import { Directive, TemplateRef, EventEmitter, QueryList, Optional, Inject } from '@angular/core';
 import { DisplayDensityBase, IDisplayDensityOptions, DisplayDensityToken } from '../core/density';
-import { CurrentResourceStrings } from '../core/i18n/resources';
-import { IListResourceStrings } from '../core/i18n/list-resources';
+// import { CurrentResourceStrings } from '../core/i18n/resources';
+// import { IListResourceStrings } from '../core/i18n/list-resources';
```

Comment:
Can we remove those instead of commenting them?

Reply:
We can and we should! Done :)